app/models/person.rb

```diff
@@ -19,6 +22,14 @@ def name
     "#{first_name} #{last_name} (#{email})"
   end
 
+  def temporary_ask_tag_list # these will be directly on the person once forms are available
+    asks.any? ? asks.map(&:tags).flatten.uniq : temporary_offer_tag_list + ["translation"]
+  end
+
+  def temporary_offer_tag_list # these will be directly on the person once forms are available
+    offers.any? ? offers.map(&:tags).flatten.uniq : ["transportation", "meals"]
+  end
+
```

Comment:
these are hack methods as a way to get the profiles populated. def will need a refactor